tomcat-job-type/app/src/main/java/Swagger.java

```diff
@@ -0,0 +1,169 @@
+                                                {numbers: [40, 2]}
+                                      responses:
+                                        '200':
+                                          description: Successful Response
+                                          content:
+                                            application/json:
+                                              schema: {}
+                                        '422':
+                                          description: Validation Error
+                                          content:
+                                            application/json:
+                                              schema:
+                                                $ref: '#/components/schemas/HTTPValidationError'
+
+                                components:
+                                  schemas:
+                                    HTTPValidationError:
+                                      properties:
+                                        detail:
+                                          items:
+                                            $ref: '#/components/schemas/ValidationError'
+                                          type: array
+                                          title: Detail
+                                      type: object
+                                      title: HTTPValidationError
+                                    ValidationError:
+                                      properties:
+                                        loc:
+                                          items:
+                                            anyOf:
+                                              - type: string
+                                              - type: integer
+                                          type: array
+                                          title: Location
+                                        msg:
+                                          type: string
+                                          title: Message
+                                        type:
+                                          type: string
+                                          title: Error Type
+                                      type: object
+                                      required:
+                                        - loc
+                                        - msg
+                                        - type
+                                      title: ValidationError
+                                  securitySchemes:
+                                    racetrackAuth:
+                                      type: apiKey
+                                      in: header
+                                      name: X-Racetrack-Auth
+                                security:
+                                  - racetrackAuth: []
+                                """;
+        String JobName = System.getenv("JOB_NAME");
```

Comment:
Indentation looks odd here. I think it's under-indented

Reply:
Actually the swagger docstring was overindented.